src/h2load.cc

```diff
@@ -669,6 +781,31 @@ void Client::on_header(int32_t stream_id, const uint8_t *name, size_t namelen,
       }
     }
 
+    if (warmup == 2) {
```

Comment:
Rather than indenting all of these, which creates more churn in the patch and visual density, I would just short circuit the function, i.e. `if (phase != SOME_STATE) { return; }`. Then you can leave much of the existing code as is.